butterfly-utilities/src/main/java/com/paypal/butterfly/utilities/xml/XmlXPathExists.java

```diff
@@ -0,0 +1,114 @@
+package com.paypal.butterfly.utilities.xml;
+
+import java.io.File;
+import java.io.IOException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathConstants;
+import javax.xml.xpath.XPathExpression;
+import javax.xml.xpath.XPathExpressionException;
+import javax.xml.xpath.XPathFactory;
+
+import org.apache.commons.lang3.StringUtils;
+import org.w3c.dom.Document;
+import org.xml.sax.SAXException;
+
+import com.paypal.butterfly.extensions.api.TUExecutionResult;
+import com.paypal.butterfly.extensions.api.TransformationContext;
+import com.paypal.butterfly.extensions.api.TransformationUtility;
+import com.paypal.butterfly.extensions.api.exception.TransformationDefinitionException;
+import com.paypal.butterfly.extensions.api.exception.TransformationUtilityException;
+
+/**
+ * Indicates whether an xpath exists in the given xml file
+ * <br>
+ * If the xpath expression won't compile, an error is returned.
+ * <br>
+ * If the file is not a well formed XML file, an error is returned.
+ *
+ * @author mmcrockett
+ */
+public class XmlXPathExists extends TransformationUtility<XmlXPathExists> {
+    private String xpathExpressionString;
+    private XPathExpression xpathExpression;
+
+    private static final String DESCRIPTION = "Retrieve the results of xml xpath query %s in XML file %s";
```

Comment:
The result is always necessarily true or false. That means an evaluation is performed, and that is why this is a utility condition. Please, see examples of other conditions and improve this message, highlight the type of evaluation that is done and what makes it returns true or false.